JS/marquee.js

```diff
@@ -0,0 +1,40 @@
+class Marquee {
+    constructor(element) {
+        this.element = element;
+        this.url = `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/quotes/nyse/`
```

Comment:
instead of rewriting the API adress, you should use global variable to hold this baseURL (https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/)

Reply:
Implemented in Milestone-6.